apps/electron-giddh/src/node-app/schemas/company.js

```diff
@@ -0,0 +1,402 @@
+        uniqueName: "string",
+        role: {
+            type: "object",
+            objectType: "Role"
+        },
+        to: "string?",
+        period: "string?",
+        allowedIps: {
+            type: "list",
+            objectType: "string"
+        },
+        allowedCidrs: {
+            type: "list",
+            objectType: "string"
+        },
+        sharedWith: {
+            type: "object",
+            objectType: "User"
+        },
+        sharedBy: {
+            type: "object",
+            objectType: "User"
+        }
+    }
+}
+
+const Company = {
+    name: "Company",
+    properties: {
+        companyTotals: "string?",
+        headQuarterAlias: "string?",
+        country: "string",
+        state: "string?",
+        uniqueName: "string",
+        city: "string?",
+        email: "string?",
+        createdBy: {
+            type: "object",
+            objectType: "User"
+        },
+        createdAt: "string",
+        addresses: {
+            type: "list",
+            objectType: "Address",
+            items: {
+                type: "object?",
+                objectType: "Address"
+            }
+        },
+        contactNo: "string?",
+        pincode: "string?",
+        updatedAt: "string?",
+        address: "string?",
+        ecommerceDetails: {
+            type: "list",
+            objectType: "Ecommerce",
+            items: {
+                type: "object?",
+                objectType: "Ecommerce"
+            }
+        },
+        baseCurrency: "string?",
+        balanceDisplayFormat: "string?",
+        businessNature: "string?",
+        updatedBy: {
+            type: "object?",
+            objectType: "User"
+        },
+        balanceDecimalPlaces: "int?",
+        businessType: "string?",
+        alias: "string?",
+        subscription: {
+            type: "object?",
+            objectType: "Subscription"
+        },
+        userEntityRoles: {
+            type: "list",
+            objectType: "UserEntityRoles",
+            items: {
+                type: "object?",
+                objectType: "UserEntityRoles"
+            }
+        },
+        nameAlias: "string?",
+        activeFinancialYear: {
+            type: "object?",
+            objectType: "FinancialYear"
+        },
+        licenceKey: "string?",
+        panNumber: "string?",
+        financialYears: {
+            type: "list",
+            objectType: "FinancialYear",
+            items: {
+                type: "object?",
+                objectType: "FinancialYear"
+            }
+        },
+        voucherVersion: "int?",
+        companyIdentity: {
+            type: "list",
+            objectType: "string"
+        },
+        baseCurrencySymbol: "string?",
+        countryV2: {
+            type: "object?",
+            objectType: "CountryV2"
+        },
+        lastAccessDetails: "string?",
+        name: "string?"
+    },
+    primaryKey: "uniqueName"
+};
+
+module.exports.User = User;
```

Comment:
Consider using for all exported Schemas:
```suggestion
module.exports = {
    User,
    Branch,
    // ...
};
```
Also at the place of import the statement used will be:
`import {User, Branch, ...} from '../path/to/module';`